autoscoper/src/ui/FilterTreeWidget.cpp

```diff
@@ -229,6 +229,22 @@ void FilterTreeWidget::saveAllSettings(QString directory){
   }
 }
 
+void FilterTreeWidget::saveAllSettings(std::ofstream& os) {
```

Comment:
Since the information appended to stream can not be used to load the settings back, I suggest to instead renamed the function to `printAllSettings`